types/react-bootstrap-typeahead/index.d.ts

```diff
@@ -0,0 +1,66 @@
+// Type definitions for react-bootstrap-typeahead 2.0
+// Project: https://github.com/ericgio/react-bootstrap-typeahead
+// Definitions by: My Self <https://github.com/Guymestef>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+import * as React from 'react';
+
+export interface TypeaheadProps<T> {
+    a11yNumResults?: () => void;
+    a11yNumSelected?: () => void;
+    align?: 'justify' | 'left' | 'right';
+    allowNew?: boolean;
+    autoFocus?: boolean;
+    bodyContainer?: boolean;
+    bsSize?: 'large' | 'lg' | 'small' | 'sm';
+    caseSensitive?: boolean;
+    clearButton?: boolean;
+    defaultSelected?: T[];
+    disabled?: boolean;
+    dropup?: boolean;
+    emptyLabel?: string;
+    filterBy?: (string[] | ((option: T | string, text: string) => boolean));
+    highlightOnlyResult?: boolean;
+    ignoreDiacritics?: boolean;
+    inputProps?: object;
+    labelKey?: string | ((option: T | string) => string);
+    maxHeight?: number;
+    maxResults?: number;
+    minLength?: number;
+    multiple?: boolean;
+    newSelectionPrefix?: string;
+    onBlur?: (e: Event) => any;
+    onChange?: (selected: T[]) => any;
+    onFocus?: (e: Event) => any;
+    onInputChange?: (input: string) => any;
+    onKeyDown?: (e: Event) => any;
+    onMenuHide?: (e: Event) => any;
+    onMenuShow?: (e: Event) => any;
+    onPaginate?: (e: Event) => any;
+    options: T[];
+    paginate?: boolean;
+    paginationText?: string;
+    placeholder?: string;
+    renderMenu?: (results: Array<T | string>, menuProps: any) => any;
+    renderMenuItemChildren?: (option: T, props: TypeaheadProps<T>, index: number) => any;
+    renderToken?: (selectedItem: T | string, onRemove: () => void) => any;
+    selected?: T[];
+    selectHintOnEnter?: boolean;
+    submitFormOnEnter?: boolean;
+}
+
+export const Typeahead: React.ClassicComponentClass<TypeaheadProps<any>>;
+
+export default Typeahead;
```

Comment:
This doesn't seem to really be a default export. The [readme](https://github.com/ericgio/react-bootstrap-typeahead) recommends using `import {Typeahead} from 'react-bootstrap-typeahead';`, which is a named import (`import Typeahead from 'react-bootstrap-typeahead';` would be a default import.)